.github/CONTRIBUTING.md

```diff
@@ -63,9 +63,9 @@ To start your local version of GitBook, run the command `bun dev`.
 
 When running the development server, published GitBook sites can be rendered through your local version at `http://localhost:3000/`.
 
-For example, our published docs can be viewed using the local version by visiting `http://localhost:3000/docs.gitbook.com` after running the development server.
+For example, our published docs can be viewed using the local version by visiting `http://localhost:3000/gitbook.com/docs` after running the development server.
```

Comment:
Were you able to test a proxied site locally like this?

Reply:
@talagluck We’ll need to test this before we go live

Reply:
@taranvohra I've just tested this and it doesn't work quite right - it redirects to the app, rather than the docs site. I compared http://localhost:3000/gitbook.com/docs with http://localhost:3000/docs.photoroom.com which works as expected.

Reply:
I've fixed that part